src/hooks/useSeen.ts

```diff
@@ -0,0 +1,65 @@
+import { useEffect, useRef } from 'react';
+
+import { selectRawIndexerHeightData } from '@/abacus-ts/selectors/base';
+import { shallowEqual } from 'react-redux';
+
+import { getUserWalletAddress } from '@/state/accountSelectors';
+import { setSeenFills, setSeenOpenOrders, setSeenOrderHistory } from '@/state/accountUiMemory';
+import { getSelectedNetwork } from '@/state/appSelectors';
+import { useAppDispatch, useAppSelector } from '@/state/appTypes';
+
+import { assertNever } from '@/lib/assertNever';
+
+export function useViewPanel(
+  market: string | undefined,
+  kind: 'fills' | 'openOrders' | 'orderHistory'
```

Comment:
would it make sense to map this to the enums used in HorizontalPanel?

Reply:
naw since only 3/5 panels track unread stuff currently